send.go

```diff
@@ -198,7 +202,9 @@ func (c *Connection) contentSummary(response *http.Response) (summary string, er
 	if err != nil {
 		return
 	}
-	runes := []rune(string(body))
+	content := strip.StripTags(string(body))
```

Comment:
This is done unconditionally but it mostly makes sense for text/html with a long body.
WDYT of checking Content-Type and/or moving this under `if len(runes) > 200`?
Not critical, pretty harmless to do it always, but in that case please update doc comment saying "summary will be the complete body if it isn't too log."

Reply:
agreed and updated. will give user raw response unless it's really messy.

Reply:
for status code and the status msg, I did have it in the old PR, https://github.com/openshift-online/ocm-sdk-go/pull/133/files#diff-6761b1a148e5dc2caa0297b163135bbcR205-R210

It was removed afterwards. It probably caused some complication. I will leave it to you to decide and add back in the future.